imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -135,31 +138,42 @@ class CaseExplorer extends Component {
   )
   render () {
     const { isLoading, caseList, allNotifications, unreadNotifications } = this.props
-    const { filterStatus, myInvolvement, open } = this.state
+    const { roleFilterValues, sortBy, open } = this.state
     if (isLoading) return <Preloader />
-    const caseGrouping = this.makeCaseGrouping(caseList, filterStatus, myInvolvement, allNotifications, unreadNotifications)
+    const caseGrouping = this.makeCaseGrouping(caseList, roleFilterValues, sortBy, allNotifications, unreadNotifications)
+    const defaultList = caseGrouping.sort(sorters[SORT_BY.LATEST_UPDATE])
+    const cases = sortBy ? caseGrouping.sort(sorters[sortBy]) : defaultList
```

Comment:
This sorting should be made in `makeCaseGrouping` and not as a part of each render method invocation